compiler/src/iree/compiler/InputConversion/StableHLO/VerifyCompilerInputLegality.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 The IREE Authors
```

Comment:
2023

Reply:
This is based on the header from the original code.  It was authored in 2021.

Reply:
I see, thanks!